packages/cozy-harvest-lib/test/components/TriggerManager.spec.js

```diff
@@ -66,7 +66,7 @@ const tMock = jest.fn()
 const client = new CozyClient({})
 const props = {
   konnector: fixtures.konnector,
-  flow: new ConnectionFlow(client),
+  flow: new ConnectionFlow(client, undefined, fixtures.konnector),
```

Comment:
Nit: At this point I am wondering if an option object would not be better, I had trouble remembering that the second argument is a trigger.